### PR TITLE
add Test_HE003_TC01,add SummaryPages,History Pages, add  TestValue

### PR DESCRIPTION
**Pull request by MelisPavol**

### CODE REVIEW

File: src/test/java/SingInTest.java [hunk 1]
```diff
@@ -38,19 +38,19 @@ void test_HE_002_TC_2_Invalid_Sing_in(){
...
+    void test_HE_002_TC_3_Valid_Sing_in_from_hamburger_menu() {
+        var appointmentPage = homePage
+                .hamburgerMenu()
+                .loginFromHamburgerMenu()
+                .enterUserName(TestValues.TEST_VALID_USERNAME)
+                .enterpassword(TestValues.TEST_VALID_PASSWORD)
+                .loginButtonOnLoginPage();
 
         assertEquals("Make Appointment", appointmentPage.getTextMakeAppointment());
     }
```
Comment (darinamas) [inline src/test/java/SingInTest.java:51]: Prosim pridej  .getTextMakeAppointment() do retezu:
var appointmentPage = homePage
                .hamburgerMenu()
                .loginFromHamburgerMenu()
                .enterUserName(TestValues.TEST_VALID_USERNAME)
                .enterpassword(TestValues.TEST_VALID_PASSWORD)
                .loginButtonOnLoginPage()
                **_.getTextMakeAppointment();_**

File: src/main/java/AppointmentPage.java [hunk 2]
```diff
@@ -40,37 +40,91 @@ public AppointmentPage(WebDriver driver) {
...
+        return this;
+    }
+    public AppointmentPage prevMount(){
+        prevMounth.click();
+        return this;
+    }
+
+    public AppointmentPage clickCommentTF(String comment){
+        //clickCommentTextField.click();
+        clickCommentTextField.sendKeys(comment);
```
Comment (darinamas) [inline src/main/java/AppointmentPage.java]: Incorrect name of the method:  "clickCommentTF"
It should be putComment, or enterComment

**Review by darinamas — CHANGES_REQUESTED** [review]

File: src/main/java/AppointmentPage.java [hunk 3]
```diff
@@ -116,14 +116,13 @@ public AppointmentPage prevMount(){
...
     }
 
-    public AppointmentPage clickCommentTF(String comment){
-        //clickCommentTextField.click();
-        clickCommentTextField.sendKeys(comment);
-       return this;
+    public AppointmentPage commentTextField(String comment){
+        commentTextField.sendKeys(comment);
+        return this;
     }
```
Comment (darinamas) [inline src/main/java/AppointmentPage.java:122]: Incorrect name of the method: "clickCommentTF"
It should be putCommentTF, or enterCommentTF

**Review by darinamas — CHANGES_REQUESTED** [review]

File: src/main/java/AppointmentPage.java (hunk 3 above)
Comment (MelisPavol) [reply src/main/java/AppointmentPage.java:122]: fixed

**Review by darinamas — APPROVED** [review]
LGFM

**Pull request merged**